lib/rules/comma-style.js

```diff
@@ -62,6 +62,49 @@ module.exports = {
         }
 
         /**
+         * Modified text based on the style
+         * @param {string} styleType Style type
+         * @param {string} text Source code text
+         * @returns {string} modified text
+         * @private
+         */
+        function getReplacedText(styleType, text) {
+            switch (styleType) {
+                case "between":
+                    return `,${text.replace("\n", "")}`;
```

Comment:
This is dangerous.

```js
let obj = {
    a // a comment
    ,
    b
}
```

I think good to handle the same way as others for lonely comma's cases also. I.e. `getFixerFunction(style, previousItemToken, commaToken, currentItemToken)`

Or maybe it should not fix the lonely commas.